pilot/pkg/ambient/ambient.go

```diff
@@ -17,13 +17,16 @@ package ambient
 type NodeType = string
 
 const (
-	LabelStatus = "istio.io/ambient-status"
-	TypeEnabled = "enabled"
-	// LabelType == "workload" -> intercept into ztunnel
-	// TODO this could be an annotation – eventually move it into api repo
-	LabelType = "ambient-type"
-
-	TypeWorkload NodeType = "workload"
-	TypeNone     NodeType = "none"
+	// AnnotationType determines the type of workload this is.
+	AnnotationType = "ambient.istio.io/type"
+	// TypeMesh indicates this workload is in the mesh and has redirection to ztunnel.
+	// This is added by the CNI when redirection is configured, not by the user.
+	TypeMesh NodeType = "mesh"
+	// TypeDisabled indicates this workload does NOT have redirection enabled. Workloads without any
+	// value should be assumed to be "disabled" as well, but explicitly setting this acts as an opt-out.
```

Comment:
```suggestion
	// value should be assumed to be "disabled" as well, but explicitly setting this acts as an opt-out.
```

Is this suggesting that an end-user might set this to `disabled`, or that there is a scenario where the CNI might (versus just not labelling at all)

Reply:
CNI will never set it to "disabled", would just not set it. "disabled" implies it was explicitly disabled

Reply:
Seems it should be: `Workloads without any value should NOT be assumed to be "disabled" as well,`